digdag-standards/src/main/java/io/digdag/standards/operator/ForRangeOperatorFactory.java

```diff
@@ -0,0 +1,113 @@
+package io.digdag.standards.operator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.base.Optional;
+import com.google.inject.Inject;
+import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigException;
+import io.digdag.core.Limits;
+import io.digdag.spi.Operator;
+import io.digdag.spi.OperatorContext;
+import io.digdag.spi.OperatorFactory;
+import io.digdag.spi.TaskRequest;
+import io.digdag.spi.TaskResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ForRangeOperatorFactory
+        implements OperatorFactory
+{
+    private static Logger logger = LoggerFactory.getLogger(ForRangeOperatorFactory.class);
+
+    @Inject
+    public ForRangeOperatorFactory()
+    { }
+
+    @Override
+    public String getType()
+    {
+        return "for_range";
+    }
+
+    @Override
+    public ForRangeOperator newOperator(OperatorContext context)
+    {
+        return new ForRangeOperator(context);
+    }
+
+    static class ForRangeOperator
+            implements Operator
+    {
+        private final TaskRequest request;
+
+        public ForRangeOperator(OperatorContext context)
+        {
+            this.request = context.getTaskRequest();
+        }
+
+        @Override
+        public TaskResult run()
+        {
+            Config params = request.getConfig();
+
+            Config doConfig = request.getConfig().getNested("_do");
+            boolean parallel = params.get("_parallel", boolean.class, false);
+
+            Config rangeConfig = params.parseNested("_command");
+            long from = rangeConfig.get("from", long.class);
+            long until = rangeConfig.get("until", long.class);
+
+            Optional<Long> stepConfig = request.getConfig().getOptional("step", Long.class);
```

Comment:
How about getting these parameters from `rangeConfig`? I think getting them from the same config would be more consistent.

Reply:
hmm...two  things: I want to avoid nested objects as much as possible because I'm seeing many people confused with YAML's indents. Another thing is that if we have `width` in nested config, it's misleading because people may think it's the width of the total range (not width of a slice).

Reply:
To avoid all syntax misunderstanding, this is the best:

```
for_range>: [min, max]  # inclusive min max
step:
slices:  # step or slices
```

Reply:
But with `step` instead of `width`, nesting seems OK.